src/scene_file.cpp

```diff
@@ -109,23 +124,31 @@ void Scene_File::Start() {
 		file_windows.push_back(w);
 	}
 
+	border_bottom = makeBorderSprite(232);
+
 	index = latest_slot;
 
 	Refresh();
 	Update();
 }
 
 void Scene_File::Refresh() {
-	for (unsigned int i = 0; (size_t) i < file_windows.size(); i++) {
+	for (int i = 0; i < int(file_windows.size()); i++) {
```

Comment:
this looks really ugly.

Reply:
Changed it to a cast. The loop variable should be signed as it is compared to signed values inside the loop.